### ISSUE TITLE
Request: Change position of notifications

### ISSUE THREAD

**Issue by tairun** [atom/notifications#93]
There should be an option to customize the anchor of the notification. Maybe vanilla atom doesn't have this problem but with the changes to the editor that i made, the notifications look like this:
![png bad](https://cloud.githubusercontent.com/assets/4248906/10118213/997ac832-6473-11e5-91fe-c13237efcd90.png)  
(Overlaping scroll bar, tree-view and tab bar)

I'd much prefer this:
![png normal](https://cloud.githubusercontent.com/assets/4248906/10118222/b59f87aa-6473-11e5-890a-4c2f6f24a7e8.png)
(Anchor top right of ``text-editor`` class).

**Comment by simurai** [comment]
This might be possible by moving it into `<atom-workspace-axis class="vertical">`, but it also feels a bit strange because it would suggest that the notifications are part of the editor and not something more global.

You could move the notifications to the left with adding the following to your `styles.less` file:

```css
atom-notifications {
  left: 0;
  right: auto;
}
```

![screen shot 2015-10-02 at 2 14 26 pm](https://cloud.githubusercontent.com/assets/378023/10239912/e890e910-690f-11e5-88b6-c09f6f427d9b.png)

Then the notifications don't cover the tree-view anymore.

**Comment by tairun** [comment]
Okay I agree that making the notification seem relevant only to the current editor can be confusing. I added your snippet to the styles.less file and flying with this for now. Added some top margin to clear tab bar :-)

(By the way is that the [https://atom.io/packages/tool-bar](tool-bar)-package in your screenshot?)

**Comment by simurai** [comment]
> Added some top margin to clear tab bar

There is the `@tab-height` that could be used, but it's not that accurate with all themes.

> (By the way is that the tool-bar package in your screenshot?)

Yes, plus https://atom.io/packages/flex-tool-bar to make it easy to add custom buttons.

**Comment by tairun** [comment]
^There is the @tab-height that could be used, but it's not that accurate with all themes.
Thanks. Where can I get a list with these veriables. Maybe theres a ``@tree-view-width`` or something.

**Comment by simurai** [comment]
Unfortunately not.. only these are officially available: https://github.com/atom/atom/blob/master/static/variables/ui-variables.less

**Comment by tairun** [comment]
Yep, just found it myself. This might be relevant too: https://github.com/atom/tree-view/issues/520

**Comment by markediez** [comment]
+1

Really would like to anchor the notifications to the bottom right...

**Comment by walderrama** [comment]
+1 for bottom right

**Comment by jens1245** [comment]
+1 for bottom right

**Comment by natedunn** [comment]
I got it to the bottom right with this quick add to styles.less

```atom-notifications {
  top:auto;
}
```

**Comment by simurai** [comment]
Moving to the bottom would cover panels like find + replace or linter etc.

![screen shot 2016-05-18 at 12 53 18 pm](https://cloud.githubusercontent.com/assets/378023/15346952/b3857d98-1cf7-11e6-8b20-368257f3d495.png)

> @tairun (Anchor top right of text-editor class).

Might still be the best option.

**Comment by code-tree** [comment]
Notifications were appearing over the file tabs for me, making me dismiss them before I could change tab. Quite annoying, so definitely vote for moving to bottom right. Or maybe bottom and whatever side the tree view is *not* on?

**Comment by damieng** [comment]
This has moved down slightly in 4c2b349bd70e121de9900fdc68789e2997ed7861 to not hit the tab area on the default themes.
